src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -82,7 +82,9 @@ public void SetGet_Roundtrips()
         [InlineData(true)]
         public void ProxySetViaEnvironmentVariable_DefaultProxyCredentialsUsed(bool useProxy)
         {
-            bool envVarsSupported = UseManagedHandler || !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            //bool envVarsSupported = UseManagedHandler || !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+	    // TODO: Issue #25640
+            bool envVarsSupported = !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
The sporadic failures you've seen have only been on Windows, never on Unix?

Reply:
yes, windows only so far.